src/charm.py

```diff
@@ -32,8 +33,8 @@
 logger = logging.getLogger(__name__)
 
 BASE_CONFIG_PATH = "/etc/smf"
-CONFIG_FILE_NAME = "smfcfg.yaml"
-UE_CONFIG_FILE_NAME = "uerouting.conf"
+CONFIG_FILE = "smfcfg.yaml"
+UE_CONFIG_FILE = "uerouting.yaml"
```

Comment:
```suggestion
UEROUTING_CONFIG_FILE = "uerouting.yaml"
```

I find it confusing having a UE configuration file in an element of the core.